bot/check-build.sh

```diff
@@ -76,6 +79,10 @@ while [[ $# -gt 0 ]]; do
       VERBOSE=1
       shift 1
       ;;
+    -a|--alt-artefacts)
```

Comment:
@smoors I would rename the option to `--use-check-build-artefacts-script`, which leaves (a lot) less room for confusion.

The `ALT_ARTEFACTS` constant should be renamed accordingly to `USE_CHECK_BUILD_ARTEFACTS_SCRIPT`